pkg/cmd/experimental/syncgroups/groupsyncer.go

```diff
@@ -102,13 +127,12 @@ func (s *LDAPGroupSyncer) updateGroup(ldapGroupUID string, usernames []string) e
 	group.Annotations[ldaputil.LDAPSyncTimeAnnotation] = ISO8601(time.Now())
 	group.Annotations[ldaputil.LDAPURLAnnotation] = s.Host
 
-	_, err = s.GroupClient.Update(group)
-	return err
+	return group, err
 }
 
-// findGroup finds the OpenShift Group for the LDAP group UID and ensures that the OpenShift Group found
+// findOpenShiftGroup finds the OpenShift Group for the LDAP group UID and ensures that the OpenShift Group found
 // was created as a result of a previous LDAP sync from the same LDAP group.
-func (s *LDAPGroupSyncer) findGroup(ldapGroupUID string) (*ouserapi.Group, error) {
+func (s *LDAPGroupSyncer) findOpenShiftGroup(ldapGroupUID string) (*userapi.Group, error) {
```

Comment:
seems like we'd want a way to force syncing to a group that doesn't match (syncing a previously manually-managed group to LDAP, or syncing a group when your LDAP server url changes, etc). @stevekuznetsov add to your LDAP punch list?